website/content/en/preview/upgrade-guide.md

```diff
@@ -91,6 +91,9 @@ By adopting this practice we allow our users who are early adopters to test out
 
 # Released Upgrade Notes
```

Comment:
I think there's other mentions of `CLUSTER_ENDPOINT` that we can get rid of in the getting started guide that we can remove in the latest preview

Reply:
I just updated all the doc positions I found reasonable. I kept the printing of the CLUSTER_ENDPOINT during cluster creations, just to have a documentation how to receive it.